Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,49 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(card) {
+    // Remove the suit from the card string
+    const rank = card.slice(0, -1);
+    
+    // Handle Number Cards (2-9)
+    if (rank >= "2" && rank <= "9") {
```

Comment:
#### Todo

This check is not solid enough. 
You can test your function by calling `getCardValue("23♠")` or `getCardValue("899♠")` to see what they return.

Reply:
I did work on this and I found a solution!
I added a set of valid ranks so only the valid ones can return an actual value